gravitee-apim-e2e/lib/utils/eventsource-fetch.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
```

Comment:
Those are not copyright from graviteee but comes from microsoft see https://github.com/Azure/fetch-event-source/tree/main/src.

You should refer to those lib or at least give the source somewhere.

Reply:
I added a reference in the top comment of the file